.github/workflows/e2e-tests.yml

```diff
@@ -398,6 +398,71 @@ jobs:
           output: |
             {"summary":${{ env.MOCHAWESOME_REPORT_RESULTS }}}
 
+  e2e-code-coverage-cypress:
+    name: E2E Code Coverage Reports
+    runs-on: ubuntu-latest
+    needs: [set-commit-sha, testing-reports-prep, cypress-tests-prep, cypress-tests]
+    continue-on-error: true
+    if: ${{ always() && needs.cypress-tests-prep.outputs.tests != '[]' && (needs.cypress-tests.result == 'success' || needs.cypress-tests.result == 'failure') }}
```

Comment:
Since we're always running all Cypress tests in this workflow, you probably won't need the `needs.cypress-tests-prep.outputs.tests != '[]'` condition here